wordfreq_builder/wordfreq_builder/word_counts.py

```diff
@@ -17,9 +17,14 @@ def count_tokens(filename):
     """
     counts = defaultdict(int)
     with open(filename, encoding='utf-8', errors='replace') as infile:
-        for line in infile:
-            for token in simple_tokenize(line.strip()):
-                counts[token] += 1
+        if lang == 'ar':
+            for line in infile:
+                for token in simple_tokenize(line):
+                    counts[standardize_arabic(token)] += 1
```

Comment:
Does it really save that much time to have the `lang == 'ar'` check outside the loop?  It certainly duplicates code over, e.g.,
```
if lang == 'ar':
    token = standardize_arabic(token)
```
Separately, though, I thought the Arabic standardizer normally operated before tokenization?  (Same comment would apply with `line = standardize_arabic(line)` instead, of course.)